File: app/src/main/java/com/inspiringteam/xchange/data/models/QuakesResponse.java
package com.inspiringteam.xchange.data.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;


public class QuakesResponse {
    @SerializedName("features")
    @Expose
    public List<QuakeWrapper> quakeWrapperList;

    public List<QuakeWrapper> getQuakeWrapperList() {
        return quakeWrapperList;
    }

    public void setquakeWrapperList(List<QuakeWrapper> quakeWrapperList) {
        this.quakeWrapperList = quakeWrapperList;
    }
}

File: app/src/main/java/com/inspiringteam/xchange/data/models/QuakesResponse.kt
package com.inspiringteam.xchange.data.models

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import java.util.*

data class QuakesResponse(@field:Expose @field:SerializedName("features") var quakeWrapperList: MutableList<QuakeWrapper>)